packages/Card/src/Card.js

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import * as sc from "./Card.styles";
+import Footer from "./components/Footer";
+import Title from "./components/Title";
+import Metadata from "./components/Metadata";
+import Header from "./components/Header";
+import Text from "./components/Text";
+import Separator from "./components/Separator";
+import Content from "./components/Content";
+
+const propTypes = {
+  /** Body content of the card. */
+  children: PropTypes.node,
+
+  /** If the width of the card should span it's parent container (100%). */
+  isFullWidth: PropTypes.bool,
+
+  /** If the height of the card should span the height of its children elements. */
+  isAutoHeight: PropTypes.bool,
+
+  /** If the card is in an "active" or "selected" state. */
+  isActive: PropTypes.bool,
+
+  /** If the card is disabled. */
+  isDisabled: PropTypes.bool,
+
+  /** Callback to be executed when the card is clicked or activated by keyboard. Typically required. */
+  onClick: PropTypes.func,
```

Comment:
🤔 I see the intention, but it seems to me that the Card shouldn't be responsible to handle this behaviour. 

What you want to do instead, is to compose the `<Card />` component with another component that we have for this kind of the scenarios the `RawButton`. So if I were a consumer I will do something like this:

```js
import RawButton from "@paprika/raw-button"

function SomeThing() {
  return (
    <RawButton onClick={() => {}}>
       <Card>{...content}</Card>
    </RawButton>
  )
}
```

thoughts?

Reply:
the `isDisabled` behaviour will also be delegated to the `raw-button`

Reply:
If we are going for this `onClick` approach, we will need to create a story to showcase this so we can test for accessibility and other things